### PR TITLE
Features added to 1st and 2nd section 

### PR DESCRIPTION
**Pull request by aveetdesai**
### Type of Change

- [x] Feature
- [x] Bug fix
- [x] Documentation

## Contributors

- @aveetdesai 
- @EstebanMendez01 
- @ColeKaupelis 

## Reminder

All GitHub Actions checks must be in a passing state in this branch or fork before this pull request can be merged.

### CODE REVIEW

**Review by burgess01 — APPROVED** [review]
This looks good! Now someone from every other team needs to approve this.

**Review by brum0505 — APPROVED** [review]

**Review by BillOchieng — APPROVED** [review]
Good Work! Let's gooooooo Man City

**Review by burgess01 — APPROVED** [review]

File: conduct.md [hunk 1]
```diff
@@ -4,35 +4,38 @@
...
 
-TODO: Write an introductory paragraph.
+It is expected that the main attitude in the class is to be encouraging and positive, towards yourself and each other. Since this class is mainly based on teams, it is very important that everyone respects each others contribution and provides constructive feedback when necessary. All students are expected to show respect to everyone both inside, and outside the classroom. Students are expected to come to class on time and prepared for collaboration.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Come to every class.
```
Comment (jnormile) [inline conduct.md]: I'd argue that incomplete phrases like those written here don't warrant a period, but I posted a message on Discord to the whole class to gauge class thoughts on the matter.

File: conduct.md [hunk 2]
```diff
@@ -4,35 +4,38 @@
...
 
-TODO: Write an introductory paragraph.
+The deadline for each task should be 12am. This will help other contributors to review and make changes where necessary before class time in order to have a sense of clarity on with problems need to be resolved or worked on. Deadlines also help with feedback that is always necessary for the project we will be doing.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Doing the work in time.
```
Comment (jnormile) [inline conduct.md]: *on time

File: conduct.md [hunk 3]
```diff
@@ -4,35 +4,38 @@
...
+- Telling the team that you finished your work.
+- Doing the work with time and do not rush things.
```
Comment (jnormile) [inline conduct.md:32]: Not sure what this line means--could you clarify?

**Review by jnormile — COMMENTED** [review]
I left a few comments within the code for suggested changes--mostly grammatical. Looks good otherwise!

File: conduct.md (hunk 1 above)
Comment (EstebanMendez01) [reply conduct.md]: fixed all of the comments.

**Review by jnormile — APPROVED** [review]
Looks good--nice improvements!

**Pull request merged**